### PR TITLE
Editing mscolab Url must be blocked on connect #637

### LINKED ISSUE #637: Editing mscolab Url must be blocked on connect
We need to disable editing of the mcolab url once connected. And enable it on disconnect.

block_editing_of_url

https://github.com/Open-MSS/MSS/blob/develop/mslib/msui/mscolab.py

### PR DESCRIPTION
**Pull request by ahelii16**
Disabled editing of the mscolab url on connecting, also disabled drop down. Enabled it again on clicking disconnect.

### CODE REVIEW

File: mslib/msui/mscolab.py [hunk 1]
```diff
@@ -163,6 +163,8 @@ def disconnect_handler(self):
         self.loginButton.setEnabled(False)
         self.addUser.setEnabled(False)
         self.connectMscolab.setEnabled(True)
+        self.url.setEnabled(True)
+        self.url.setEditable(True)
```
Comment (Tanish0019) [inline mslib/msui/mscolab.py]: Not sure but do we need to use setEditable as well? Won't just using setEnabled(False) be enough as it completely disabled the input?

Comment (ahelii16) [reply mslib/msui/mscolab.py]: Yes you're right. I did not think of that. Wouldn't need line 167 as well if setEnabled is True!

**Review by Tanish0019 — APPROVED** [review]

**Comment by Tanish0019** [comment]
@ReimarBauer Approved this, you can check and merge it

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
wait, I see it is not the wrong number it is the wrong text, I fix it

**Review by ReimarBauer — APPROVED** [review]

**Pull request merged**